From 67666b2d01cbbf9154e936f3c8856a636c5492b7 Mon Sep 17 00:00:00 2001
From: Ming Ma <mingma@apache.org>
Date: Wed, 15 Feb 2017 20:24:07 -0800
Subject: [PATCH 2313/2748] HDFS-11265. Extend visualization for Maintenance
 Mode under Datanode tab in the NameNode UI.
 (Marton Elek via mingma)

(cherry picked from commit a136936d018b5cebb7aad9a01ea0dcc366e1c3b8)
(cherry picked from commit 0f8b99fd347284dad0c95c2040e46223bc42fed0)

Change-Id: I03b0e9a5029bb9824876906efca6f019ffff84cc
---
 .../hadoop-hdfs/src/main/webapps/static/hadoop.css |    9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/hadoop.css b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/hadoop.css
index 0c80eec..e2b6655 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/hadoop.css
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/hadoop.css
@@ -236,8 +236,8 @@ header.bs-docs-nav, header.bs-docs-nav .navbar-brand {
 }
 
 .dfshealth-node-decommissioned:before {
-    color: #eea236;
-    content: "\e136";
+    color: #bc5f04;
+    content: "\e090";
 }
 
 .dfshealth-node-down:before {
@@ -250,6 +250,11 @@ header.bs-docs-nav, header.bs-docs-nav .navbar-brand {
     content: "\e017";
 }
 
+.dfshealth-node-down-maintenance:before {
+    color: #eea236;
+    content: "\e136";
+}
+
 .dfshealth-node-legend {
     list-style-type: none;
     text-align: right;
-- 
1.7.9.5

